bucket/fnproject.json

```
@@ -1,5 +1,6 @@
 {
     "homepage": "https://fnproject.io/",
+    "description": "An open-source container-native serverless platform that you can run anywhere -- any cloud or on-premise.",
```

I personally do not like adress user directly in description. Rather have neutral description.

```suggestion
    "description": "A container-native serverless platform that can be runned anywhere (any cloud or on-premise).",
```